app/assets/stylesheets/modern-business.scss

```diff
@@ -241,3 +247,33 @@ footer {
   font-size: 40px;
   text-align: center;
 }
+
+/* Countdown Clock */
+
+#clockdiv{
```

Comment:
Avoid using id selectors<br>Opening curly brace `{` should be preceded by one space